tests/acceptance/app-uses-helpers-test.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../../tests/helpers/start-app';
+import {
+  clickOn,
+  clickRole,
+  fillInField,
+  findRole,
+} from 'ember-tb-test-helpers';
+
+module('Acceptance | App uses helpers', {
+  beforeEach() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).